pypeit/core/flux.py

```diff
@@ -407,16 +793,15 @@ def load_standard_file(std_dict):
     if std_dict['fmt'] == 1:
         std_spec = fits.open(fil)[1].data
         # Load
-        std_dict['wave'] = std_spec['WAVELENGTH']*units.AA
-        std_dict['flux'] = 1e17*std_spec['FLUX']*units.erg/units.s/units.cm**2/units.AA
+        std_dict['wave'] = std_spec['WAVELENGTH'] * units.AA
+        std_dict['flux'] = 1e17 * std_spec['FLUX'] * units.erg / units.s / units.cm ** 2 / units.AA
```

Comment:
I'm not sure why we are adding all of these spaces.
I guess they are harmless..

Reply:
harmless indeed